apps/blockchain/contracts/starknet/src/bridge/bridge.cairo

```diff
@@ -0,0 +1,57 @@
+use starknet::ContractAddress;
+
+#[abi]
+trait IBridgableToken {
+    fn permissioned_mint(to: ContractAddress, token_id: u256);
+}
+
+#[contract]
+mod Bridge {
+    use starknet::ContractAddress;
+    use starknet::contract_address_const;
+    use starknet::get_caller_address;
+    use starklane_contracts::bridge::escrow_bridge::IBridgeEscrowDispatcherTrait;
+    use starklane_contracts::bridge::escrow_bridge::IBridgeEscrowDispatcher;
+    use super::IBridgableTokenDispatcherTrait;
+    use super::IBridgableTokenDispatcher;
+
+    struct Storage {
+        _l1_to_l2_addresses: LegacyMap::<ContractAddress, ContractAddress>,
+        escrow_contract: ContractAddress
+    }
+
+
+    #[constructor]
+    fn constructor(_escrow_contract: ContractAddress) {
+        escrow_contract::write(_escrow_contract);
+    }
+
+
+    #[view]
+    fn read_l2_address(l1_address: ContractAddress) -> ContractAddress {
+        _l1_to_l2_addresses::read(l1_address)
+    }
+
+    #[external]
+    fn mint_token(
+        l1_address: ContractAddress, l2_recipient_address: ContractAddress, token_id: u256
+    ) {
+        let l2_contract_address: ContractAddress = _l1_to_l2_addresses::read(l1_address);
+        if (l2_contract_address != contract_address_const::<0>()) {
```

Comment:
It's better to use Zeroable when possible in cairo. `ContractAddressZeroable::zero()` in the case of `ContractAddress`.

Reply:
Which module should I "import" to use ContractAddressZeroable::zero() ?